### PR TITLE
SDN-3681: Fix SDN migration jobs

### PR DESCRIPTION
**Pull request by pliurh**
Remove egressip configuration after finishing the egressip validation.

In AWS, the egressip might result in it appearing in the node status
as an additional internal IP. It will break the nodeip service tests.
Since the tests will use the egressip as the nodeip mistakenly.

### CODE REVIEW

**Comment by pliurh** [comment]
/pj-rehearse pull-ci-openshift-cluster-network-operator-master-e2e-network-migration-aws-ovn

**Comment by pliurh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-network-migration-rollback

**Comment by pliurh** [comment]
@jluhrsen PTAL

**Comment by pliurh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.13-e2e-network-migration-rollback

**Comment by kyrtapz** [comment]
@pliurh could you add an explanation as to why the removal of the egressips was required?
slack thread for context: https://redhat-internal.slack.com/archives/GQ0CU2623/p1674064265181409

**Comment by pliurh** [comment]
> @pliurh could you add an explanation as to why the removal of the egressips was required? slack thread for context: https://redhat-internal.slack.com/archives/GQ0CU2623/p1674064265181409

Done.

File: ci-operator/jobs/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1929,9 +1929,9 @@ presubmits:
       ci-operator.openshift.io/cloud-cluster-profile: aws
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-priv-cluster-network-operator-master-e2e-sdn-network-migration-rollback
+    name: pull-ci-openshift-priv-cluster-network-operator-master-e2e-sdn-network-migration-rollback-aws-sdn
```
Comment (danwinship) [inline ci-operator/jobs/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master-presubmits.yaml:1932]: so it's got "sdn" in the name twice now...

**Comment by danwinship** [comment]
/approve
/hold
if people think the name should be fixed to not say "sdn" twice. feel free to `/hold cancel` if not.

**Comment by jluhrsen** [comment]
> /approve /hold if people think the name should be fixed to not say "sdn" twice. feel free to `/hold cancel` if not.

good point. I agree.

@pliurh, the pattern most jobs follow are e2e-<platform>-<cni>-<other tags>

can you rename the jobs so they would match that? so e2e-aws-sdn-network-migration?

**Comment by pliurh** [comment]
I updated the job name. PTAL.

**Pull request merged**